upgrade/1.0.1/scripts/upgrade/add_pod_priority.sh

```diff
@@ -147,10 +151,14 @@ for statefulset in $STATEFULSETS; do
   if [ ! -z "$ns" ]; then
     echo "Patching $statefulset statefulset in $ns namespace"
     kubectl -n $ns patch statefulset $statefulset --type merge -p '{"spec": {"template": {"spec": {"priorityClassName": "csm-high-priority-service"}}}}'
+    kubectl rollout status statefulset -n $ns $statefulset
     echo ""
   fi
 done
 
+echo "Creating a backup of cray-bss etcdcluster prior to restarting the cluster"
+kubectl exec -it -n operators $(kubectl get pod -n operators | grep etcd-backup-restore | head -1 | awk '{print $1}') -c util -- create_backup cray-bss pod-priority-backup-$(date "+%D-%T") | grep -v 'unknown operand'
```

Comment:
An alternate way to get the pod name using just kubectl:
```
kubectl get pod -n operators -l app.kubernetes.io/name=cray-etcd-backup -o jsonpath='{.items[0].metadata.name}'
```

Reply:
Note: this is just a suggestion, no change needed.

Reply:
good suggestion, will fix